packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -299,8 +299,8 @@ contract VotingV2 is
                 // The voter voted correctly. Receive a pro-rate share of the other voters slashed amounts as a reward.
             else
                 slash += int256(
-                    (((voterStake.cumulativeStaked * 1e18) / requestSlashingTrackers[i].totalCorrectVotes) *
-                        requestSlashingTrackers[i].totalSlashed) / 1e18
+                    (((voterStake.cumulativeStaked * requestSlashingTrackers[i].totalSlashed)) /
```

Comment:
this is as a result of you forking off your other PR, right? so we need to merge that one first.